docs/USAGE.rst

```diff
@@ -0,0 +1,366 @@
+
+After all images and Excel metadata files have been processed, a message is displayed summarizing what images were in each of the five classifications above (e.g., "Import completed. 3 images added. 1 DeID Upload Excel file parsed. See the Excel file report for more details.). If you click on the "See the Excel report for more details" link, it will download an import report.
+
+Below is a screenshot of a message presented to the user after an import.
+
+.. image:: screenshots/import_message_highlighted.png
+   :height: 100
+   :width: 200
+   :alt: import message
+
+Exporting Data
+==============
+
+When images are in the ``WSI DeID`` collection, in the ``Approved`` folder, they can be exported out of the DSA WSI DeID for transfer. 
+
+In the ``Approved`` folder, two buttons appear at the top: ``Export Recent`` and ``Export All``, as shown in the screenshot below. Clicking either copies files from the ``Approved`` folder to the mounted export folder, that is, to the local filesystem folder that was mounted as the export path in the docker-compose configuration. The subfolder structure within the ``Approved`` folder is maintained as part of the export. If a file already exists in the export folder, then that file will be skipped during the export process so as to not overwrite the existing file in the export directory. 
+
+.. image:: screenshots/export_buttons_highlighted.png
+   :height: 100
+   :width: 200
+   :alt: export buttons
+
+Recent exports are any items in the ``Approved`` folder that have not been exported before. After each export, items are tagged with metadata indicating that they have been exported.
+
+After export, a message is shown indicating how many files were exported, how many were already present (based on having the same file name) and the same file size, and how many were already present and differed in file size.
+
+
+Redaction
+=========
+
+At a high level in the Redaction process, the user can inspect the image and metadata for PHI/PII, and can indicate if any of these should be redacted. Redaction can occur from the ``Available To Process`` or ``Quarantined`` folder. Below are the specific steps to follow for the Redaction process. When a field is redacted so that it is changed, the original value in that redacted data field is completely replaced with the new value. When a field or image is redacted completely, it is removed and will not be available in the future.
+
+1. Reviewing the image(s) and metadata for PHI/PII.
+
+  1.1 Open the WSI file, either by clicking on a specific Item or clicking the ``Next Item`` action.
+
+  1.2 Review the image(s) and metadata fields for PHI/PII.
+
+    1.2.1 Review the image. If the low-resolution image at the top of the screen (see screenshot below) contains PHI/PII, then click XXX. If there is no PHI/PII, then go on to the next step. You may zoom and pan in the low-resolution image to see areas of the image in higher resolution.
```

Comment:
I would refer to this as the "zoomable image" rather than the low-resolution image.  If there is PHI/PII, click reject.